src/components/dialogHelper/dialogHelper.js

```diff
@@ -375,7 +375,7 @@ import 'scrollStyles';
             dlg.setAttribute('data-lockscroll', 'true');
         }
 
-        if (options.enableHistory !== false && appRouter.enableNativeHistory()) {
+        if (options.enableHistory !== false) {
```

Comment:
This broke menu (see @nyanmisaka post). Dialogs+history is too fragile combination

Reply:
revert `appRouter.enableNativeHistory()` indeed fix that.